src/messages/SharedMessageBuilder.cpp

```diff
@@ -30,6 +30,34 @@ namespace {
         }
     }
 
+    QStringList parseTagList(const QVariantMap &tags, const QString &key)
```

Comment:
Duplicate of TwitchMessageBuilder's `parseTagList` - fine to keep duplicate for now